auth/auth.go

```diff
@@ -108,16 +108,30 @@ func (a *app) Consolef(format string, args ...interface{}) {
 	}
 }
 
-type claim struct {
-	Iss           string      `json:"iss"`
-	Sub           string      `json:"sub"`
-	Aud           interface{} `json:"aud"` // since it depends on the scopes if aud is a string or []string
-	Exp           int         `json:"exp"`
-	Iat           int         `json:"iat"`
-	AtHash        string      `json:"at_hash"`
-	Email         string      `json:"email"`
-	EmailVerified bool        `json:"email_verified"`
-	Name          string      `json:"name"`
+type Claims struct {
+	Id              string            `json:"jti,omitempty"`
+	ExpiresAt       int64             `json:"exp,omitempty"`
+	IssuedAt        int64             `json:"iat,omitempty"`
+	NotBefore       int64             `json:"nbf,omitempty"`
+	Issuer          string            `json:"iss,omitempty"`
+	Subject         string            `json:"sub,omitempty"`
+	Audience        interface{}       `json:"aud,omitempty"`
+	Groups          []string          `json:"groups"`
```

Comment:
is groups here needed ?

Reply:
yes, this struct contains all possible claims for dex and generic tokens